middlewares/validators/user.js

```diff
@@ -16,10 +16,8 @@ const updateUser = async (req, res, next) => {
       username: joi
         .string()
         .optional()
-        .min(4)
-        .max(20)
-        .regex(/^[a-zA-Z0-9]+$/)
-        .message("Username must be between 4 and 20 characters long and contain only letters or numbers."),
+        .regex(/^[a-z0-9-]+$/)
```

Comment:
Hello @Abhay5855, I want that username validator should enforce the specific format of usernames
i.e. Usernames should start with first name,  followed by a hyphen, then the last name.
ex:  username validator enforces the required username format of "firstname-lastname" or "firstname-lastname-1"

Is it possible
Like this below regex
<img width="334" alt="image" src="https://github.com/Real-Dev-Squad/website-backend/assets/107163260/44b4444e-56d1-40d3-bb0a-b120c0de3b94">

So if hypen is before firstname it is invalid username or number before firstname is invalid username

Reply:
@shubhamsigdar1 That's great thinking I have made the validation as required such that firstname-lastname-1 or firstname-lastname can be included rest will throw an error.